www/base/src/styles/styles.less

```diff
@@ -74,6 +74,9 @@
   }
 }
 
+.label {
```

Comment:
I just did a quick test on nine.bb.net with the release tarball, and it looks okay. I did test that the UI is even better if you change the ``span.label`` by a ``a.label(href="#")`` because there is a mouse over effect on top of the cursor change